sdks/python/apache_beam/transforms/core.py

```diff
@@ -1608,10 +1608,18 @@ def __init__(self, windowfn, **kwargs):
 
     Args:
       windowfn: Function to be used for windowing
+      trigger: (optional) Trigger used for windowing, or None for default.
+      accumulation_mode: Accumulation mode used for windowing,
```

Comment:
Should this also be marked optional? It seems optional looking at the code.

Reply:
It's sometimes optional. Updating the comment.